config/packages/workflow.yaml

```diff
@@ -28,3 +28,100 @@ framework:
+        type: method
+        property: status
+      supports:
+        - SprykerSdk\SdkContracts\Entity\WorkflowInterface
+      initial_marking: start
+      places:
+        - start
+        - manifest
+        - config
+        - translation
+        - acp-validated
+        - openapi
+        - openapi-validated
+        - asyncapi
+        - asyncapi-validated
+        - asyncapi-generate
+        - pbc-validate
+        - pbc-validated
+      transitions:
+        CreateManifest:
+          from: start
+          to: manifest
+          metadata:
+            task: acp:manifest:create
+            name: BuildManifest
+        CreateConfiguration:
+          from: manifest
+          to: config
+          metadata:
+            task: acp:config:create
+        CreateTranslation:
+          from: config
+          to: translation
+          metadata:
+            task: acp:translation:create
+        ValidateACP:
+          from: translation
+          to: acp-validated
+          metadata:
+            task: acp:app:validate
+            error: Now you need to fix manually and continue with the workflow afterwards.
+        CreateOpenApi:
+          from: acp-validated
+          to: openapi
+          metadata:
+            task: acp:openapi:create
+        ValidateOpenApi:
+          from: openapi
+          to: openapi-validated
+          metadata:
+            task: acp:openapi:validate
+            error: Now you need to fix manually and continue with the workflow afterwards.
+        GenerateOpenApi:
+          from: openapi-validated
+          to: pbc-validate
+          metadata:
+            task: acp:openapi:generate
+        CreateAsyncapi:
+          from: pbc-validate
+          to: asyncapi
+          metadata:
+            task: acp:asyncapi:create-full
+        ValidateAsyncapi:
+          from: asyncapi
+          to: asyncapi-validated
+          metadata:
+            task: acp:asyncapi:validate
+            error: Now you need to fix manually and continue with the workflow afterwards.
+        GenerateAsyncapiCode:
+          from: asyncapi-validated
+          to: asyncapi-generate
+          metadata:
+            task: acp:asyncapi:generate
+        ValidatePbc:
+          from: pbc-validate
+          to: pbc-validated
+          metadata:
+            task: acp:pbc:validate
+            error: Now you need to fix manually and continue with the workflow afterwards.
+        ValidatePbcWithAsyncApi:
+          from: asyncapi-generate
+          to: pbc-validated
+          metadata:
+            task: acp:pbc:validate
+            error: Now you need to fix manually and continue with the workflow afterwards.
+            name: ValidatePbc
+        RevalidationPbc:
+          from: pbc-validated
+          to: pbc-validated
+          metadata:
+            task: acp:pbc:validate
+            error: Now you need to fix manually and continue with the workflow afterwards.
+            name: ValidatePbc
+
```

Comment:
Where are all other steps defined in https://spryker.atlassian.net/wiki/spaces/CORE/pages/2990604664/PBC+Development ?

The ones defined in the Validation section are missing.

Reply:
because it's task set type.
```- id: "validation:php:codestyle-fix"
    stop_on_error: true
  - id: "validation:php:codestyle-check"
    stop_on_error: true
  - id: "validation:php:static"
    stop_on_error: true
  - id: "validation:php:architecture"
    stop_on_error: true
  - id: "tests:sdk:acceptance"
    stop_on_error: true```